src/sdk/p2p/peerconnection-channel.js

```diff
@@ -646,14 +646,23 @@ class P2PPeerConnectionChannel extends EventDispatcher {
     const pc = new RTCPeerConnection({
       sdpSemantics: 'unified-plan',
     });
+    if (typeof pc.getConfiguration === "undefined") {
+      return false;
+    }
     return (pc.getConfiguration() && pc.getConfiguration().sdpSemantics ===
```

Comment:
since you've already check validity of getConfiguration above, don't do it again.